bower.json

```diff
@@ -25,7 +25,8 @@
     "nunjucks": "nunjucks#1.0.5",
     "nwmatcher": "nwmatcher#1.3.4",
     "pikaday": "~1.3.2",
-    "react": "#0.13.0",
+    "react": "#0.13.3",
```

Comment:
We don't have a specific convention, but it is generally good to keep PRs as small as small as reasonable. If you aren't using anything specific to 0.13.3, I think it makes sense to move this to a separate PR.